db/migrate/20240517145354_create_sign_in_certificates.rb

```diff
@@ -0,0 +1,15 @@
+class CreateSignInCertificates < ActiveRecord::Migration[7.1]
+  def change
+    create_table :sign_in_certificates do |t|
```

Comment:
Is it valuable to have any of the attribute fields other than the certificate itself? I feel like the schema shouldn't have any fields that are already available on the cert itself, otherwise there could be user error in the stored attributes versus what's actually on the cert

Reply:
Having the fields stored in columns on the table allows us to leverage the ORM to query for expired, expiring, and self-signed certificates more efficiently.

The alternative would be to instantiate a 509 certificate object for every certificate stored in the table and then iterate through them to find all the certificates that match whatever criteria there are (expired, expiring, self-signed, etc.) This is the approach I used in my [original PR](https://github.com/department-of-veterans-affairs/vets-api/pull/16733/files).

I'd go so far as to say if we don't store the fields in columns, then this refactor is probably unnecessary.